cpp/src/arrow/util/bit_util_benchmark.cc

```diff
@@ -30,8 +33,6 @@ namespace arrow {
 
 namespace BitUtil {
 
-#ifdef ARROW_WITH_BENCHMARKS_REFERENCE
```

Comment:
You shouldn't remove this, should you?

Reply:
Ah, I removed that while debugging. I'll revert this